llvm/test/Transforms/LoopVectorize/AArch64/partial-reduce-dot-product.ll

```diff
@@ -218,41 +218,50 @@ define i64 @dotp_i8_to_i64_has_neon_dotprod(ptr readonly %a, ptr readonly %b) #1
 ; CHECK-INTERLEAVE1-LABEL: define i64 @dotp_i8_to_i64_has_neon_dotprod(
```

Comment:
I think it would be good to add `not` to the start of the function name to match the convention with the other functions. Same for the i16 to i64 one.